monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/TypeFilter.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import SelectComponent from '../Select';
+import _ from 'lodash';
+
+type TypeFilterProps = {
+  allPlugins: any[],
+  displayedPlugins: any[],
+  setDisplayedPlugins: (plugins: any[]) => void,
+  className?: string
+}
+
+type SelectOption = {
+  value: string,
+  label: string
+}
+
+const anyTypeOption :SelectOption = {value: "", label: "All"}
+
+const TypeFilter = (props: TypeFilterProps) => {
```

Comment:
Destruct props

Reply:
Don't think it's worth it here. It's easier to tell what comes from props and what's a local symbol

Reply:
It will be more readable + it's better for setting default values.